src/themes/base/siteVariables.ts

```diff
@@ -8,11 +8,13 @@ export { colors, contextualColors, emphasisColors, naturalColors, colorScheme }
 // FONT SIZES
 //
 export const fontSizes = {
+  smallest: '0.8rem',
```

Comment:
while it seems to be logical, the counterpoint is that we, generally, will 'close' the interval of possible values in terms of extensibility - especially when we are talking about `base` theme. The problem is that, with this being, essentially, a finite set, we cannot guarantee that client's scenario won't require additional sizes added to the set - and, in that case, we cannot ensure that client will be able to extend `base` theme and introduce, say, 10 (and not 7, as prescribed) font sizes _with meaningful names_ . 

So, because of that I am a bit reluctant to introduce this change now.